_scripts/update.py

```diff
@@ -49,18 +49,22 @@ def add_config(fx_root: str, fx_cfg_path: str, done: set[str]):
             cfg = tomllib.load(file)
         cfg["basepath"] = ".."
         for path in cfg["paths"]:
+            if path["reference"].endswith("/locales/en-US/**"):
+                paths.add(path["reference"].replace("/en-US/**", ""))
```

Comment:
This will miss out paths like `.../locales/en-US/foo/**`, but in practice those are almost certain to be covered by corresponding `.../locales/en-US/**` paths elsewhere.

Reply:
The reason I went this way is [mobile](https://searchfox.org/mozilla-central/source/mobile/android/locales/l10n.toml), because replacing wouldn't work there. But I realize now that I can't just truncate after `/en-US`, just to be safe.

Reply:
Changed logic in https://github.com/mozilla-l10n/firefox-l10n-source/pull/20/commits/439c780724c7254020efaa4fd2d13bbf0b50888f? Is that what you had originally in mind in the issue comment?

Reply:
Yeah, something like that. This filter does not need to be perfect, just inclusive of all the things that it's supposed to include. If we do end up with say, 1000 extra files that we ultimately don't need, even that's unlikely to slow things down notably.